scripts/use-react-version.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
```

Comment:
This file is basically a rename of `use-reaect-dist-tag.js` with some changes, but for some reason GitHub is not picking it up. I tried to use `git mv` too, and it was shown as a rename in VS code, but not here.

Reply:
>  I tried to use git mv too, and it was shown as a rename in VS code, but not here.

Git does not have a notion of moving files. `git mv` is really just a (cross-os compatible) wrapper around `mv old new; git add old; git add new`.

The difference you're seeing is purely due to difference in [`--find-renames`](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---find-renamesltngt)

If you want to make review easier. Split rename and change into different commits so that people can review on a per-commit basis.

Reply:
Aha, so the issue is not renaming & changing in the same comment. I will keep this in mind in the future.